packages/excubiae/contracts/IExcubia.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+/// @title IExcubia
+/// @notice Interface for the excubia contract.
+interface IExcubia {
```

Comment:
I think you could remove the event from the `Excubia` contract and inherit from the Interface instead (so you also ensure that the abstract contract adheres to the interface) - this way you could also move all errors to the interface

Reply:
ooo thank you, I totally missed it!

please, see 66625e003ab73dad80ba39510b790c482c239772